agreement_rappel/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2020 Tecnativa - Carlos Dauden
+# Copyright 2020 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    'name': 'Agreement Rappel',
```

Comment:
Shouldn't be 'Agreement Reminder' instead as rappel is for climbing ... (and the french translation for reminder) ?

Reply:
Sorry, We thought that the English term was rappel. 

As @JordiBForgeFlow comments here:
https://github.com/OCA/commission/issues/238#issuecomment-626144706
the correct term should be "rebate":
https://en.wikipedia.org/wiki/Rebate_(marketing)

Do you agree?

Reply:
@rousseldenis Are you agree with the name??